libraries/ms-common/test/test_adc.c

```diff
@@ -0,0 +1,70 @@
+#include "adc.h"
+#include "gpio.h"
+#include "unity.h"
+#include "log.h"
+
+static GPIOAddress address[] = {
```

Comment:
Please use port enums and I doubt enumerating every single address is necessary.